R/radae.R

```diff
@@ -164,6 +164,13 @@ radae <- function(ADSL, # nolint
       origin = "1970-01-01"
     )) %>%
     dplyr::mutate(AENDY = ceiling(as.numeric(difftime(.data$AENDTM, .data$TRTSDTM, units = "days")))) %>%
+    dplyr::mutate(LDOSEDTM = as.POSIXct(
+      ifelse(.data$TRTSDTM < .data$ASTDTM,
+        lubridate::as_datetime(runif(1, .data$TRTSDTM, .data$ASTDTM), tz = "EST"),
```

Comment:
hi @edelarua , I was wondering if this is the timezone we use everywhere?

Reply:
@shajoezhu yes, all of the dates in this dataset use the Eastern time zone.

Reply:
Excellent! Thanks for checking this @edelarua